avy.el

```diff
@@ -427,7 +427,8 @@ KEYS is the path from the root of `avy-tree' to LEAF."
           ((mouse-event-p char)
            (signal 'user-error (list "Mouse event not handled" char)))
           (t
-           (signal 'user-error (list "No such candidate" char))
+           (signal 'user-error (list "No such candidate"
+                                     (if (characterp char) (string char) char)))
```

Comment:
This change isn't necessary. The idea here is that the calling code can catch the error and have `char` ready for use.

Reply:
But maybe they don't and then you get this error which shows a number
instead of a character.  Not very helpful.  I have experienced this many
times in working with the ace-window code where this error is not caught.